packages/storybook/stories/Localization.stories.mdx

```diff
@@ -8,3 +21,86 @@ Localization is a key to making products that can be used across the world and b
 UI Library will provide out of the box support for some languages and capabilities such as RTL.
 Developers can provide their own localization files to be used for the UI Library.
 These localization capabilities will be added ahead of General Availability.
+
+## LocalizationProvider
+
+`LocalizationProvider` is a wrapping component that sets all the strings for ACS components using a `Locale`. We have a set of
```

Comment:
nit: We have a set of locals -> Out of the box, the UI library includes a set of locales usable with the UI components and composites.